js/react_native/ios/OnnxruntimeModule.mm

```diff
@@ -8,6 +8,10 @@
 #import <React/RCTLog.h>
 #import <onnxruntime/onnxruntime_cxx_api.h>
 
+#ifdef ENABLE_ORT_EXT
+#include <onnxruntime_extensions.h>
```

Comment:
so this include doesn't require any leading directory?

Reply:
correct, this one works fine. it's the <onnxruntime_c_api.h>  this header includes requires a leading directory.

Reply:
update: actually looks like just getting rid of https://github.com/microsoft/onnxruntime-extensions/blob/0e2d6c03e4b87f49e6d35c1ca540e99317ac8bb9/includes/onnxruntime_extensions.h#L5
this line here would work with the project. (i.e. no "redefinition" error at compile stage for objects defined in onnxruntime_c_api.h)

I guess this include isn't really necessary maybe.

Reply:
@wenbingl what do you think? could we remove the onnxruntime_c_api.h include from [here](https://github.com/microsoft/onnxruntime-extensions/blob/0e2d6c03e4b87f49e6d35c1ca540e99317ac8bb9/includes/onnxruntime_extensions.h#L5) and just forward declare the ORT types?

Reply:
> Does this require a change on the ort extensions side for it to work? If so are there any objections to that change?

yeah... hopefully we can just remove that line as Ed pined above.

Reply:
> @wenbingl what do you think? could we remove the onnxruntime_c_api.h include from [here](https://github.com/microsoft/onnxruntime-extensions/blob/0e2d6c03e4b87f49e6d35c1ca540e99317ac8bb9/includes/onnxruntime_extensions.h#L5) and just forward declare the ORT types?

onnxruntime_extensions.h is the only ABI interface for ort-extensions, and Register... function depends on some types defined c_api.h. Can we assume that every time the onnxruntime_extensions.h is included, onnxruntme_c_api.h already included?

I don't know what the issue is here? redefinition? Suppose there will be only one copy of onnxruntime_c_api.h in the whole build since ort-extensions can use onnxruntime_c_api.h from internal folder, if not, that's something should be fixed.

Reply:
> I don't know what the issue is here? redefinition? Suppose there will be only one copy of onnxruntime_c_api.h in the whole build since ort-extensions can use onnxruntime_c_api.h from internal folder, if not, that's something should be fixed.

yea, I experienced a redefinition error when building a react native app on the ios side. I guess it duplicates with the onnxruntime_c_api.h header that already been included from the onnxruntime-c pod. 

I will try to seek ways to not making changes on the ort-extensions side first.

Reply:
The ort header file are setting here: https://github.com/microsoft/onnxruntime-
extensions/blob/f12270b06858b2a57c6fa5041c39d835a707060f/cmake/ext_ortlib.cmake#L2 
You may make some change here if needed to make sure the right header file can be found.